app/models/dto.py

```
@@ -67,6 +67,75 @@ class AvailabilityRequest(BaseModel):
+    @field_validator('capacity')
+    @classmethod
+    def validate_capacity_range(cls, v: int) -> int:
+        if not (1 <= v <= 100):
+            raise ValueError("인원은 1명 이상 100명 이하여야 합니다.")
+        return v
+
+    @model_validator(mode='after')
+    def validate_logic(self) -> 'AvailabilityRequest':
+        # 1. Coordinate Logic
+        if self.swLat >= self.neLat:
+            raise ValueError("남서쪽 위도(swLat)는 북동쪽 위도(neLat)보다 작아야 합니다.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)